checker.cpp

```diff
@@ -1,46 +1,87 @@
 #include <iostream>
 #include <assert.h>
 #include <unistd.h>
+#include <map>
 using namespace std;
 
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if(temperature > 102 || temperature < 95) {
-    cout << "Temperature critical!\n";
+string language = "German"; // global variable to set the language
+//Contains Transltons...
+map<string, map<string, string>> translations = {
+    {"English", {
+        {"Temperature critical!", "Temperature critical!"},
```

Comment:
The left side ('key' in the map) is the actual string. Consider using neutral strings. For example, if you get a feedback to remove the exclamation mark from the string, you will need to remove in multiple places and change the code, or keep it on the left and remove on the right. Not a big deal, but a bit confusing got people who maintain